Mykhailo_Honcharov/assignment1.py

```diff
@@ -0,0 +1,94 @@
+def main() -> None:
+    
+    # String Analysis
+
+    print("Hello!")
+    first_name = input("What is your first name?")
+    last_name = input("What is your last name?")
```

Comment:
```suggestion
    first_name = input("What is your first name? ")
    last_name = input("What is your last name? ")
```

give me some space, man